examples/step-71/doc/results.dox

```diff
@@ -0,0 +1,167 @@
+  </tr>
+</table>
+We can see that the $L_2$ norm convergence rates are around 5,
+$H_1$-seminorm convergence rates are around 4,
+and $H_2$-seminorm convergence rates are around 3.
+On the finest mesh, the $L_2$ norm convergence rate
+is much smaller than our theoretical expectations
+because the linear solver becomes the limiting factor due
+to round-off. But the $L_2$ error is pretty small in that case.
+
+<h3>Test results on <i>Q<sub>2</sub></i> with <i>&gamma; = 1</i> </h3>
+
+<h4>Convergence table</h4>
+
+
+<table align="center" class="doxtable">
+  <tr>
+   <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H_1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H_2}$ </th><th> Conv. rates </th>
+  </tr>
+  <tr>
+   <td>   2                  </td><td>    4.86048e-02 </td><td>       </td><td>   3.30386e-01   </td><td>           </td><td>  4.34917 </td><td>   </td>
+  </tr>
+  <tr>
+   <td>   3                  </td><td>    1.29921e-02   </td><td>  1.90  </td><td> 1.4852e-01   </td><td>    1.15    </td><td> 4.01192  </td><td>  0.116 </td>
+  </tr>
+  <tr>
+   <td>   4                  </td><td>   3.33539e-03   </td><td>  1.96    </td><td>  7.20252e-02    </td><td>  1.04       </td><td>  3.96138 </td><td> 0.018  </td>
+  </tr>
+  <tr>
+   <td>   5                  </td><td>    8.41058e-04 </td><td>  1.98   </td><td>  3.57705e-02  </td><td>    1.00       </td><td> 3.95719  </td><td>  0.001   </td>
+  </tr>
+</table>
+Although $L_2$ norm and $H_1$-seminorm convergence rates of $u$
+follow the theoretical expectations, $H_2$-seminorm does not converge.
+Comparing results from $\gamma = 1$ and $\gamma = p(p+1)$, it is clear that
+$\gamma = p(p+1)$ is a better penalty.
+
+
+<h3>Test results on <i>Q<sub>2</sub></i> with <i>&gamma; = 2</i> </h3>
+
+<h4>Convergence table</h4>
+
+
+<table align="center" class="doxtable">
+  <tr>
+   <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H_1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H_2}$ </th><th> Conv. rates </th>
+  </tr>
+  <tr>
+   <td>   2                  </td><td>    5.482e-03 </td><td>       </td><td>   7.652e-02  </td><td>           </td><td>  1.756e-00 </td><td>   </td>
+  </tr>
+  <tr>
+   <td>   3                  </td><td>    2.227e-02   </td><td>  1.29  </td><td> 2.177e-02   </td><td>   1.81   </td><td> 8.711e-01  </td><td> 1.01 </td>
+  </tr>
+  <tr>
+   <td>   4                  </td><td>  9.088e-04   </td><td>  1.29    </td><td> 6.026e-03    </td><td>  1.85       </td><td> 4.196e-01 </td><td> 1.05  </td>
+  </tr>
+  <tr>
+   <td>   5                  </td><td>   2.822e-04  </td><td>  1.68  </td><td> 1.605e-03 </td><td>    1.90     </td><td> 2.041e-01  </td><td> 1.03  </td>
+  </tr>
+</table>
+In the case, all convergence rates of $u$
+follow the theoretical expectations.
```

Comment:
I don't understand. We don't see a rate of 2 for L2, which we expect.

Reply:
I would guess this is a suboptimal parameter choice and we should say so (or delete this section).